src/scripts/control-plane.sh

```diff
@@ -45,7 +44,13 @@ kubectl apply -f /src/manifests/metrics-server/
 kubectl -n kube-system scale deployment coredns --replicas=1
 
 # get admin token
-kubectl describe secret "$(kubectl get secrets | grep cluster | cut -d ' ' -f1)" | grep token:  | tr -s ' ' | cut -d ' ' -f2 > /src/output/cluster-admin-token
+until kubectl get secrets -o name | grep cluster
+do
+    echo "Waiting for admin secret to be created"
```

Comment:
isn't creted at line 12 during `kubeadm init`?

Reply:
Yes, but it is taking a while until available in kubectl output, hence this while loop.